### PR TITLE
feat(query): add new query for tencentcloud CDB resource

### PR DESCRIPTION
**Pull request by SevenEarth**
Hello, There is something new query for TencentCloud TKE resource:

cdb_instance_internet_service_enabled
cdb_instance_using_default_intranet_port
cdb_instance_without_backup_policy

Please help review the PR
Thanks

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Comment by SevenEarth** [comment]
This is my local test
image

**Comment by SevenEarth** [comment]
Hello
Please help review the PR
Thanks

File: assets/queries/terraform/tencentcloud/cdb_instance_using_default_intranet_port/query.rego [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	resource := input.document[i].resource.tencentcloud_mysql_instance[name]
+    not common_lib.valid_key(resource, "intranet_port")
+
+	result := {
+		"documentId": input.document[i].id,
+		"resourceType": "tencentcloud_mysql_instance",
+		"resourceName": tf_lib.get_resource_name(resource, name),
+		"searchKey": sprintf("tencentcloud_mysql_instance[%s]",[name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": sprintf("[%s] 'intranet_port' should be set and the value cannot byt 3306",[name]),
```
Comment (gabriel-cx) [inline assets/queries/terraform/tencentcloud/cdb_instance_using_default_intranet_port/query.rego:32]:
```suggestion
		"keyExpectedValue": sprintf("[%s] 'intranet_port' should be set and the value should not be 3306",[name]),
```

File: assets/queries/terraform/tencentcloud/cdb_instance_without_backup_policy/query.rego [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	resource := input.document[i].resource.tencentcloud_mysql_instance[name]
+    not any_backup_policy_matches_instance(name)
+
+	result := {
+		"documentId": input.document[i].id,
+		"resourceType": "tencentcloud_mysql_instance",
+		"resourceName": tf_lib.get_resource_name(resource, name),
+		"searchKey": sprintf("tencentcloud_mysql_instance[%s]", [name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": sprintf("tencentcloud_mysql_instance[%s] should has 'tencentcloud_mysql_backup_policy'", [name]),
```
Comment (gabriel-cx) [inline assets/queries/terraform/tencentcloud/cdb_instance_without_backup_policy/query.rego:16]:
```suggestion
		"keyExpectedValue": sprintf("tencentcloud_mysql_instance[%s] should have 'tencentcloud_mysql_backup_policy'", [name]),
```

**Review by gabriel-cx — CHANGES_REQUESTED** [review]

**Comment by gabriel-cx** [comment]
@SevenEarth thanks for the contribution! kindly take a look into the code review.

File: assets/queries/terraform/tencentcloud/cdb_instance_internet_service_enabled/query.rego [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	resource := input.document[i].resource.tencentcloud_mysql_instance[name]
+    resource.internet_service == 1
+
+	result := {
+		"documentId": input.document[i].id,
+		"resourceType": "tencentcloud_mysql_instance",
+		"resourceName": tf_lib.get_resource_name(resource, name),
+		"searchKey": sprintf("tencentcloud_mysql_instance[%s].internet_service", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("[%s] has 'internet_service' set to 0 or undefiend", [name]),
```
Comment (ArturRibeiro-CX) [inline assets/queries/terraform/tencentcloud/cdb_instance_internet_service_enabled/query.rego:16]:
```suggestion
		"keyExpectedValue": sprintf("[%s] has 'internet_service' set to 0 or undefined", [name]),
```

**Review by ArturRibeiro-CX — CHANGES_REQUESTED** [review]
Hey @SevenEarth, thanks for your contribution so far! 
Kindly review the requested changes please.

**Review by ArturRibeiro-CX — APPROVED** [review]

**Comment by SevenEarth** [comment]
> 嘿@SevenEarth，感谢您迄今为止的贡献！ 请检查所请求的更改。

Thank you all, I will notice this issue in the next development

**Pull request merged**